pages/events.tsx

```diff
@@ -70,34 +70,44 @@ const EventsPage = () => {
       })
   }, [setLoading])
 
+  //  gets today's date in the user's timezone, in ISO format (YYYY-MM-DD)
+  const today = new Date().toLocaleDateString("sv");
```

Comment:
so there's no built-in way to use `toISODateString()` with the user's timezone. so I used this somewhat hacky approach from [this stackoverflow answer](https://stackoverflow.com/a/60368477)

I had to consider timezone, since using 'toISODateString()` on its own sets `today` to one day ahead (UTC is ahead of PST).

Reply:
Also, the reason ISO format is important is because it allows easy date comparisons, since it works with the way javascript compares strings.

Reply:
great